resources/js/processes/modeler/validateFlowGenieRef.js

```diff
@@ -0,0 +1,16 @@
+const validateFlowGenieRef = () => ProcessMaker.EventBus.$on("modeler-validate", (node, reporter) => {
+  if (node.$type !== "bpmn:ServiceTask" || node.implementation !== "package-ai/processmaker-ai-task") {
+    return;
+  }
+
+  if (node.config) {
+    const config = JSON.parse(node.config);
```

Comment:
`JSON.parse` could throw an exception if `node.config` is invalid, consider to catch this potential exception.